### ISSUE TITLE
(MEDIUM) Fix dark mode issues for tBTC

### ISSUE THREAD

**Issue by liz-shinn** [threshold-network/token-dashboard#435]
## Compilation of Dark Mode issues
_Click on each of the following to expand them for screenshots. There are Figma links for each of these to the dark mode versions in the design._

<details>
<summary> 1. Color contrast issues on the note when you first click into the bridge page. Fix colors and padding to match the design. Missing "x" close option.</summary

IMPLEMENTATION:

<img width="1433" alt="Screenshot 2023-02-03 at 3 43 28 PM" src="https://user-images.githubusercontent.com/57226633/216722516-b352dc72-ad4a-456a-a8d5-81f6e731a3b1.png">

DESIGN:

https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7884%3A207983&t=DHgw39NKOIwyALV4-1
<img width="731" alt="Screenshot 2023-02-03 at 5 22 55 PM" src="https://user-images.githubusercontent.com/57226633/216722551-4ca39f20-54e3-4268-8872-9ce969144e70.png">
</details>

<details>
<summary>2. Color contrast issue with the purple on the righthand timeline panel.</summary>

IMPLEMENTATION

<img width="260" alt="Screenshot 2023-02-03 at 5 27 21 PM" src="https://user-images.githubusercontent.com/57226633/216723007-675ffb72-8e58-4f24-a46f-dc89429fe491.png">

DESIGN

https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7207%3A199760&t=DHgw39NKOIwyALV4-1
<img width="197" alt="Screenshot 2023-02-03 at 5 28 11 PM" src="https://user-images.githubusercontent.com/57226633/216723078-a3683ed8-c6d6-4a54-b984-767bcf695135.png">
</details>

<details>
<summary>3. Color contrast issues and padding for dark mode banner.</summary>

IMPLEMENTATION

<img width="1020" alt="Screenshot 2023-02-03 at 5 36 10 PM" src="https://user-images.githubusercontent.com/57226633/216724214-3cb6065d-7d82-469e-a17e-4919ef4f8d6e.png">

DESIGN

https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7884%3A208322&t=DHgw39NKOIwyALV4-1
<img width="782" alt="Screenshot 2023-02-03 at 5 35 59 PM" src="https://user-images.githubusercontent.com/57226633/216724275-43d439b1-b8ae-45ea-9c84-2b9861ce7b18.png">
</details>

<details>
<summary>4. How it Works page color contrast on all illustrations. Make sure to swap out brand purple for dark mode purple.</summary>

IMPLEMENTATION

<img width="499" alt="Screenshot 2023-02-03 at 5 37 30 PM" src="https://user-images.githubusercontent.com/57226633/216724377-fac497df-f8a0-4160-90c8-6809d02fde13.png">
<img width="481" alt="Screenshot 2023-02-03 at 5 37 34 PM" src="https://user-images.githubusercontent.com/57226633/216724379-c7565386-11c0-4a16-8b45-7adeb76d6bd3.png">
<img width="455" alt="Screenshot 2023-02-03 at 5 37 36 PM" src="https://user-images.githubusercontent.com/57226633/216724381-8402e59b-5659-41f4-9062-ec8e12b97680.png">

DESIGN

https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7884%3A208620&t=DHgw39NKOIwyALV4-1
<img width="287" alt="Screenshot 2023-02-03 at 5 53 07 PM" src="https://user-images.githubusercontent.com/57226633/216726129-c5f035af-98b0-436d-9833-f86e93366441.png">
<img width="254" alt="Screenshot 2023-02-03 at 5 53 09 PM" src="https://user-images.githubusercontent.com/57226633/216726131-bdc77508-1903-4699-b324-89be4e384ec0.png">
<img width="293" alt="Screenshot 2023-02-03 at 5 53 13 PM" src="https://user-images.githubusercontent.com/57226633/216726132-6a4fc0a2-507d-409e-8b54-e88c2b5b07bb.png">
</details>

<details>
<summary>5. The telescope background web shape some lines are broken -- they don't match the design.</summary>

IMPLEMENTATION

<img width="380" alt="Screenshot 2023-02-03 at 5 28 33 PM" src="https://user-images.githubusercontent.com/57226633/216723308-f4ade0ac-44f1-4ae8-bb59-a0410b4b003e.png">

DESIGN

https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7207%3A199760&t=DHgw39NKOIwyALV4-1
<img width="353" alt="Screenshot 2023-02-03 at 5 30 14 PM" src="https://user-images.githubusercontent.com/57226633/216723333-a60791d0-0729-4c7f-b8af-1779fdc9e211.png">
</details>

<details>
<summary>6. Must swap out this purple for a lighter color in dark mode text -- this is unreadable.</summary>

IMPLEMENTATION

<img width="336" alt="Screenshot 2023-02-03 at 5 56 06 PM" src="https://user-images.githubusercontent.com/57226633/216726467-094793d9-161e-472e-8bd5-91dae93d7823.png">

DESIGN

https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7207%3A199760&t=DHgw39NKOIwyALV4-1
<img width="663" alt="Screenshot 2023-02-03 at 5 56 58 PM" src="https://user-images.githubusercontent.com/57226633/216726515-feff5271-68c3-489f-8f16-74d75d0ee356.png">
</details>

<details>
<summary>7. Unreadable text on the "Initiate minting tBTC" modal. </summary>

IMPLEMENTATION
![Screenshot 2023-02-06 at 4 02 26 PM](https://user-images.githubusercontent.com/57226633/217099569-562bea5c-0a7f-4a65-8a88-d293f62804ff.png)
![Screenshot 2023-02-06 at 4 02 34 PM](https://user-images.githubusercontent.com/57226633/217099572-a924df26-9b2a-42cd-8d91-6208a780e9fd.png)

Solution: (PENDING) Darken the background color of modals to be grey-800.
</details>

<details>
<summary>8. Dark mode illustrations should be added here in dark mode. This is the timeline of all steps and the JSON modal. </summary>
These are not the dark mode illustrations. They can be found in the Figma link earlier in this thread related to these images.

IMPLEMENTATION
![Screenshot 2023-02-06 at 3 59 44 PM](https://user-images.githubusercontent.com/57226633/217102991-376fb2bf-4450-4494-910b-c97e69d2336b.png)
<img width="839" alt="Screenshot 2023-02-06 at 4 48 14 PM" src="https://user-images.githubusercontent.com/57226633/217103030-e705825b-3a5e-4495-b2dd-72dbd70ed723.png">
![Screenshot 2023-02-06 at 4 00 14 PM](https://user-images.githubusercontent.com/57226633/217102958-897cfc89-0620-49d3-a920-28575020ecde.png)

DESIGN
https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7937%3A212794&t=4TUanGGKUXZcN0kF-1
![Screenshot 2023-02-07 at 10 26 39 AM](https://user-images.githubusercontent.com/57226633/217289017-f520b195-efe8-4315-9be9-dd9153662e34.png)

</details>

<details> 
<summary> 9. Legibility issue of text on BTC success screen </summary>

IMPLEMENTATION
<img width="517" alt="Screenshot 2023-02-06 at 5 37 13 PM" src="https://user-images.githubusercontent.com/57226633/217103355-d6f7d080-a5d9-46dd-bb3f-8e6d653d524e.png">

DESIGN
https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7935%3A209433&t=2win3PhJMjLCAlak-1
<img width="655" alt="Screenshot 2023-02-06 at 5 44 59 PM" src="https://user-images.githubusercontent.com/57226633/217104499-f3edb7b1-5bfc-4a6d-b34f-a99446a49ef5.png">
</details>

<details> 
<summary> 10. Legibility issue of purple & text on Step 3 page </summary>
IMPLEMENTATION

![Screenshot 2023-02-06 at 4 01 36 PM](https://user-images.githubusercontent.com/57226633/217291195-75a7a40a-2bb9-4cdf-946e-0aa755be66f2.png)

DESIGN
https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7340%3A200418&t=2win3PhJMjLCAlak-1
<img width="633" alt="Screenshot 2023-02-07 at 11 06 55 AM" src="https://user-images.githubusercontent.com/57226633/217298708-4beb76b9-9d80-487d-b739-3d3bf779b416.png">

</details>

<details> 
<summary> 11. Legibility issue of dark mode on Step 2 page and QR code </summary>

IMPLEMENTATION

<img width="529" alt="Screenshot 2023-02-06 at 5 48 12 PM" src="https://user-images.githubusercontent.com/57226633/217104979-b0c73131-4bcf-4308-b684-25c19d6ddff4.png">

DESIGN
https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7340%3A199427&t=2win3PhJMjLCAlak-1
<img width="615" alt="Screenshot 2023-02-07 at 11 08 05 AM" src="https://user-images.githubusercontent.com/57226633/217299010-b17a9804-fdd1-4564-9539-38d83b9a74f8.png">

</details>

<details> 
<summary> 12. Protocol History gradient </summary>

![image](https://github.com/threshold-network/token-dashboard/assets/10741774/cf2e030d-ec8e-455f-8146-2979f87a41c1)

</details>

**Comment by liz-shinn** [comment]
@pdyraga Added a few more dark mode issues and the ones that @sashatanase noted in Discord.
